examples/deepzoom/deepzoom_server.py

```diff
@@ -125,6 +129,8 @@ def tile(slug, level, col, row, format):
             abort(404)
         try:
             tile = app.slides[slug].get_tile(level, (col, row))
+            if app.config["USE_ICC"]:
```

Comment:
You'd need to add a default on line 59 so this doesn't fail if we're run as a WSGI application that doesn't go through the `if __name__ == "__main__"` block.

Probably the setting should be named starting with `DEEPZOOM_` for consistency with the other settings.